### DISCUSSION TITLE
PriorMC ensemble numbers

### DISCUSSION THREAD

**Discussion by laat0003** [gmdsi/GMDSI_notebooks#101] [Q&A] [answered]
Hi,

When discussing priorMC in the Australian session yesterday I forgot to ask about ensemble realization numbers for investigating data conflict. I'm trying to ascertain if the ideal number of realizations are a function of the number of parameters. In other words, is it possible for observations to be flagged as conflicting with the prior due to inadequate exploration of said prior - thereby triggering a false positive? 

Cheers,
Tariq

**Comment by rhugman** [comment, marked as answer]
Conceptually, I'd guess it is possible to flag a false positive - but, if thats happening, and assuming you have a reasonable prior ensemble size, then your prior and likelihood distributions are just barely overlapping...intuitively, that probably still suggests something is off (with the prior, the measurement or the model...).

One can always test at what number of realisations the prior variance of an obs converges by iteratively increasing the number of reals. If you are still getting PDC...then it is likely not a false positive.

**Comment by jtwhite79** [comment]
I'd also add that even with a few tens of realizations, I think you can get a pretty good picture of at least the first and second moments of the model outputs.  And if the observed value is within 2-3 (or even 4) standard deviations of the mean of those outputs, then I think that's "in the noise" and I'd accept that.  But what I think we are more concerned with is a huge distance between the outputs and observed value - such a large disparity that its obvious that no number of realizations is going close they gap.  Its this situation that yields very large residuals (that have same sign for all realizations) that can cause the ensemble to collapse on extreme (combinations of) parameter values.

**Comment by laat0003** [comment]
Thanks for both responses. It helps clear things up a lot. I'm currently working on a project at a location with strong topographic control of the water-table, which has observations over a large hydraulic head range in close proximity to one another. I'm adopting the method recommended in the GMDSI problem decomposition document and have been sinking a huge amount of time into re-gridding and reparametrizing so that the prior at least captures the observations. I didn't really have a feel for "how close is good enough" when it came to the residuals if I wasn't capturing the observation.

Cheers,
Tariq
